package.json

```diff
@@ -26,10 +26,12 @@
     "prettier:fix": "npm run prettier -- --write",
     "format": "npm run prettier:fix && npm run lint:fix",
     "scripts:release-post": "NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
-    "test": "cross-env NODE_NO_WARNINGS=1 jest --passWithNoTests",
-    "test:watch": "npm run test -- --watch",
-    "test:ci": "npm run test -- --coverage",
-    "test:snapshot": "npm run test -- --updateSnapshot",
+    "test": "npm run test:unit && npm run test:storybook",
+    "test:unit": "cross-env NODE_NO_WARNINGS=1 jest --passWithNoTests",
+    "test:unit:watch": "npm run test:unit -- --watch",
+    "test:storybook": "npx concurrently -P -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"npm run storybook -- --ci\" \"npx wait-on http://localhost:6006 && npx test-storybook {@}\"",
+    "test:storybook:snapshot": "npm run test:storybook -- -- --updateSnapshot",
```

Comment:
Maybe also add a `test:unit:snapshot` for all individual unit tests with snapshots? (might exist)